items.json

```diff
@@ -108784,14 +108784,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_medium_goedendag_v1_h0",
-    "baseId": "crpg_medium_goedendag_v1",
+    "id": "crpg_medium_goedendag_v2_h0",
```

Comment:
speed reduction is OK but should be offset by an increase in damage to match tiers